nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -87,6 +87,8 @@
 
 - `nodePackages.ganache` has been removed, as the package has been deprecated by upstream.
 
+- `titaniumenv`, `titanium`, and `titanium-alloy` have been removed due to lack of maintenance.
```

Comment:
```suggestion
- `titaniumenv`, `titanium`, and `titanium-alloy` have been removed due to lack of maintenance in Nixpkgs.
```

Reply:
It seems upstream development is still ongoing, so we should specify it's an issue on our end